docs/dev/lsp-extensions.md

```diff
@@ -656,6 +656,31 @@ interface TestInfo {
 }
 ```
 
+## Hover Range
+
+**Issue:** https://github.com/microsoft/language-server-protocol/issues/377
```

Comment:
This should have `**Experimental Server Capability:** { "hoverRange": boolean }`, see how thouse are defiened for some other requests in this file.

Reply:
You'll want to add an actual capability to https://github.com/rust-analyzer/rust-analyzer/blob/3a59b56324db99475ea2027cb94f82785b16ad14/crates/rust-analyzer/src/caps.rs#L115-L125